ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -12,6 +10,48 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# nanmedian
+@handle_frontend_test(
+    fn_tree="numpy.nanmedian",
+    dtype_and_a=statistical_dtype_values(function="nanmedian"),
+    dtype=helpers.get_dtypes("float", full=False, none=True),
+    where=np_frontend_helpers.where(),
+    keep_dims=st.booleans(),
+)
+def test_numpy_nanmedian(
+    dtype_and_a,
+    dtype,
+    where,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+    keep_dims,
+):
+    input_dtypes, a, axis = dtype_and_a
+    if isinstance(axis, tuple):
+        axis = axis[0]
+    where, input_dtypes, test_flags = np_frontend_helpers.handle_where_and_array_bools(
+        where=where,
+        input_dtype=input_dtypes,
+        test_flags=test_flags,
+    )
+    np_frontend_helpers.test_frontend_function(
+        input_dtypes=input_dtypes,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        a=a[0],
+        axis=axis,
+        dtype=dtype[0],
+        out=None,
+        keepdims=keep_dims,
+        where=where,
```

Comment:
your function does not have a "where" argument, so no need to generate it.

Reply:
It has where as an input

Reply:
but your function is like this (def nanmedian(a, /, *, axis=None, keepdims=False, out=None, overwrite_input=False):) right?

Reply:
so, the original function (nanmedian) does not have any problems right? the problem nao in test nanmedian only right?

Reply:
I suppose so.